pom.xml

```diff
@@ -66,7 +66,7 @@
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-core</artifactId>
-      <version>1.651</version>
+      <version>2.7.1</version>
```

Comment:
The biggest change...  😁

Reply:
Regrettably needed to get the Declarative dependency.